airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -1430,4 +1442,32 @@ private void runAndCheckWithoutNormalization(final List<AirbyteMessage> messages
     retrieveRawRecordsAndAssertSameMessages(catalog, messages, getDefaultSchema(config));
   }
 
+  protected void deserializeNestedObjects(List<AirbyteMessage> messages, List<AirbyteRecordMessage> actualMessages) {
```

Comment:
can you add a javadoc explaining what this method does? I'm having trouble understanding it. Seems like `actualMessages` is supposed to contain json objects of `{"key": "serializedJsonBlob"}`, and this method converts them to `{"key": {realJsonContents...}}`?

Reply:
done